pytest-selenium/tests/test_attribution.py

```diff
@@ -0,0 +1,39 @@
+from selenium import webdriver
+import unittest
```

Comment:
imports are not ordered correctly. they should be like the following:

```
# standard library first
import unittest  # import x first
from time import sleep  # from x import y second

# pip installed things second separated by line break
import pytest   # import x first 
from selenium import webdriver  # from x import y second
```
This takes a bit of experience and practice. The order is standard library imports like `unittest` and `time` are first. The selenium module is installed via pip so it is second separated by a line break. Within those there is another order. The ones with `import x` are first with `from x import y` being last. Within their little groups they are sorted alphabetically.